style.css

```diff
@@ -199,6 +199,321 @@ body {
+  flex-direction: column;
+  margin-bottom: 45px;
+  box-shadow: 20px 40px 40px -48px;
+}
+
+.second-page__title {
+  font: 700 45px 'Montserrat-Bold', sans-serif;
+  letter-spacing: -1px;
+  margin-bottom: 11px;
+}
+
+.second-page__subtitle {
+  font: 600 16px 'Montserrat-Bold', sans-serif;
+  letter-spacing: 2.3px;
+  margin-bottom: 67px;
+  opacity: .5;
+}
+
+.second-page__description {
+  font: 600 16px/23px 'Montserrat-Regular', sans-serif;
+  letter-spacing: .1px;
+  margin-bottom: 45px;
+  opacity: .3;
+}
+
+.second__options {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.second-page__color-options {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 55px;
+}
+
+.eclipse {
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  cursor: pointer;
+  margin-right: 14px;
+}
+
+.eclipse-white {
+  background-color: #dedede;
+}
+
+.eclipse-pink {
+  transform: scale(1.2);
+  background-color: #f29f9f;
+}
+
+.eclipse-blue {
+  background-color: #a8bff9;
+}
+
+.eclipse-green {
+  background-color: #90bd8f;
+}
+
+.eclipse-white:hover,
+.eclipse-blue:hover,
+.eclipse-green:hover {
+  transform: scale(1.1);
+}
+
+.eclipse-pink:hover {
+  transform: scale(1.4);
+}
+
+.second-page__price {
+  margin-right: 7px;
+  margin-bottom: 53px;
+  font: 700 45px 'Montserrat-Bold', sans-serif;
+}
+
+.second-page__quantity {
+  border: #000 solid 1.5px;
+  padding: 5px;
+}
+
+.second__minus {
+  padding: 10px;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.second__minus:hover {
+  background-color: #fae8e4;
+  border-radius: 20%;
+}
+
+.second__quantity__input {
+  text-align: center;
+  padding: 10px;
+  width: 15px;
+}
+
+.second__plus {
+  padding: 10px;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.second__plus:hover {
+  background-color: #fae8e4;
+  border-radius: 20%;
+}
+
+.second__add-to-cart {
+  font: 500 14px 'Montserrat-Medium', sans-serif;
+  padding: 11px 44px;
+  border: 1.5px solid #000;
+  background-color: #fff;
+  cursor: pointer;
+  width: 275px;
+}
+
+.second__add-to-cart:hover {
+  box-shadow: 0 0 0 4px #000;
+  text-decoration: underline;
+  transition: .3s ease;
+}
+
+.second__more-items {
+  font: 500 14px 'Montserrat-Medium', sans-serif;
+  color: #000;
+  cursor: pointer;
+  text-decoration: none;
+  letter-spacing: .1px;
+  border-bottom: 2px solid transparent;
+  padding-left: 5px;
+}
+
+.second__more-items:hover {
+  border-bottom: 2px solid #000;
+  transition: .3s ease;
+}
+
+.second__more-vector {
+  padding-left: 14px;
+  width: 48%;
+}
+
+/* Третья странца */
+
+.third-page {
+  background: linear-gradient(to right, #ebe3d8 422px, #000 422px, #000 502px, #ebe3d8 502px);
+  min-height: 981px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.third-page__img {
+  padding-left: 96px;
+}
+
+.third-page__img-clothes {
+  width: 109%;
+  padding-top: 5px;
+  box-shadow: -2px 32px 37px -20px rgba(0,0,0,0.2);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)